lib/razorpay/item.rb

```diff
@@ -0,0 +1,34 @@
+require 'razorpay/request'
+require 'razorpay/entity'
+
+module Razorpay
+  # Item API allows you to create and fetch customers on Razorpay
+  class Item < Entity
+    def self.request
+      Razorpay::Request.new('items')
+    end
+
+    def self.create(options)
+      request.create options
+    end
+
+    def self.fetch(id)
+      request.fetch id
+    end
+
+    def self.edit(id, options = {})
+      if(!options.is_a?(String) && options.key?(:active))
+        options[:active] = (options[:active] ? 1 : 0)
+      end 
+      request.patch id, options
```

Comment:
Updating url is `item/{id}`

Reply:
its already there https://github.com/razorpay/razorpay-ruby/blob/item_update/lib/razorpay/item.rb#L8 while requesting it append

Reply:
According to doc, its `item/{id}`. What you have implemented is `items/{id}`

Reply:
<img width="576" alt="Screenshot 2022-04-06 at 5 39 38 PM" src="https://user-images.githubusercontent.com/89454448/161971489-630ca473-0517-41d6-b0b5-397837bf4270.png">
i checked that . the doc need to be correction. if you check in code snippet `curl` its `items/{id}`

Reply:
need to confirm link